app/services/meta-tags.ts

```diff
@@ -53,10 +53,15 @@ export interface LinkMetaTagAttrs {
     href: string;
 }
 
-export type MetaTagAttrs = NameMetaTagAttrs | PropMetaTagAttrs | LinkMetaTagAttrs;
+export interface ScriptTagAttrs {
+    type: Content;
+}
+
+export type MetaTagAttrs = NameMetaTagAttrs | PropMetaTagAttrs | LinkMetaTagAttrs | ScriptTagAttrs;
 
 export interface HeadTagDef {
     type: string;
+    content?: object;
```

Comment:
Hey Yuhuai, no, we want an object here. If we need a meeting to cover this, please add one.

Reply:
Ah I see why it did not work for you. Let's do a deep dive here:

Since you are testing locally against the endpoint by asking it to return data in `turtle` format, the response returned from the server has `Content-Type: text/turtle`. If you ask the server to return data in `google-dataset-json-ld` format, the response returned from the server would then have `Content-Type: application/ld+json`.

The reason this slight distinction in response header could throw things off is because `authenticatedAjax` uses jquery `ajax()` under the hood. JQuery would try to detect the Content-Type of the response, and parse the data accordingly: if the response is in `text/turtle`, then the resolved value would be a `string`; if the response is in `application/ld+json`, then the resolved value would be a POJO. It is true that POJO means "plain old JavaScript object", and in this case it should be! Because the resolved value should be an object that contains just data and no other methods when the response is in `application/ld+json`.

The difference between the resolved value types of these two types of requests is what causes the weird behavior that you exemplified in the screenshots above: when the value is already a string, calling `JSON.stringify()` on it would return a string that simply wraps the original string with double quotes in it. (e.g. `JSON.stringify('a')` would give you `' "a" '`). You can try this in the browser yourself.

And that's what is happening in your screenshot. You try to call `JSON.stringify()` on what's already a string and then pass that string to the `content`, causing it to render with quotation marks wrapped around. When you pass the "un-stringified" string to the `content` field, it is only normal that it renders correctly, because you are already passing a string to it, because it expects one.

That is why I said `content` is a `string` and not an `object`. Your screenshot actually gives you clue as to why it is a `string` and not an `object`. I hope I am explaining things clearly. Tell me if you have any more question.

Reply:
Hey, thank you Yuhuai. That does help. Currently, it is returning the endpoint's object (either JSON-LD or Turtle) and placing that as the content in between the two script tags. When stringifying the object returned, using your suggestion, it will result in the string version shown in the screenshots. Are you requesting for the string version or an object in the tags? Google shows the head tag as holding an object, not a string, but would like to clarify.

Reply:
Ok, per our discussion and testing this is now formatted as a string.